pkgs/applications/networking/instant-messengers/webcord/default.nix

```diff
@@ -56,36 +53,34 @@ buildNpmPackage rec {
     install -Dm644 sources/assets/icons/app.png $out/share/icons/hicolor/256x256/apps/webcord.png
 
     # Add xdg-utils to path via suffix, per PR #181171
-    makeWrapper '${electron_25}/bin/electron' $out/bin/webcord \
-      --prefix LD_LIBRARY_PATH : ${libPath}:$out/opt/webcord \
+    makeWrapper '${lib.getExe electron_25}' $out/bin/webcord \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libpulseaudio pipewire ]}:$out/opt/webcord \
       --suffix PATH : "${lib.makeBinPath [ xdg-utils ]}" \
       --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}" \
       --add-flags $out/lib/node_modules/webcord/
 
     runHook postInstall
   '';
 
-  desktopItems = [
-    (makeDesktopItem {
-      name = "webcord";
-      exec = "webcord";
-      icon = "webcord";
-      desktopName = "WebCord";
-      comment = meta.description;
-      categories = [ "Network" "InstantMessaging" ];
-    })
-  ];
+  desktopItems = makeDesktopItem {
+    name = "webcord";
+    exec = "webcord";
+    icon = "webcord";
+    desktopName = "WebCord";
+    comment = "A Discord and SpaceBar electron-based client implemented without Discord API";
+    categories = [ "Network" "InstantMessaging" ];
+  };
 
   passthru.updateScript = nix-update-script { };
 
-  meta = with lib; {
+  meta = {
     description = "A Discord and SpaceBar electron-based client implemented without Discord API";
     homepage = "https://github.com/SpacingBat3/WebCord";
     downloadPage = "https://github.com/SpacingBat3/WebCord/releases";
     changelog = "https://github.com/SpacingBat3/WebCord/releases/tag/v${version}";
-    license = licenses.mit;
+    license = lib.licenses.mit;
     mainProgram = "webcord";
-    maintainers = with maintainers; [ huantian ];
-    platforms = platforms.linux;
+    maintainers = with lib.maintainers; [ huantian ];
+    platforms = lib.platforms.linux;
```

Comment:
Isn't it standard practice to use `with lib;` for `meta`? Even though not the best practice in general, it's often used for meta.

Reply:
it used to be, but recently nixpkgs has been moving from `with lib;` to explicit `lib.value` for everything

Reply:
Ah I see, is there a PR/issue for this that I should take a look at?

Reply:
I don't have a link at hand but I *think* there was an issue regarding the `with lib;` usage here in nixpkgs.

Reply:
https://github.com/NixOS/nixpkgs/issues/208242